examples/angular/src/pages/ui/components/authenticator/reset-password/reset-password.component.ts

```diff
@@ -1,23 +1,14 @@
-import { Component, OnInit } from '@angular/core';
-import { Amplify, I18n } from 'aws-amplify';
-import { translations } from '@aws-amplify/ui-angular';
+import { Component } from '@angular/core';
+import { Amplify } from 'aws-amplify';
 
 import awsExports from './aws-exports';
 
 @Component({
   selector: 'reset-password',
   templateUrl: 'reset-password.component.html',
 })
-export class ResetPasswordComponent implements OnInit {
+export class ResetPasswordComponent {
   constructor() {
     Amplify.configure(awsExports);
   }
-
-  ngOnInit() {
-    I18n.putVocabularies(translations);
-    I18n.setLanguage('en');
-    I18n.putVocabulariesForLanguage('en', {
-      'Send Code': 'Update Information',
-    });
```

Comment:
Angular in particular had a lot of digression in its resetPassword screen, and we were testing it separately. Now it is aligned with other implementation, and we can remove these custom tests with the default one.